### PR TITLE
Added new Coverage option: Clear the Coverage output directory

### PR DESCRIPTION
**Pull request by IgorSalevan**
- Added new option: Clear the Coverage output directory;
- Bumped the minor version: 0.3.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ Requirement: `npm install nyc`
 | --coverage, --cov          | Enable coverage measurement                           | `boolean` | `false`                         |
 | --coverage-instrument-spec | Coverage instrument pattern from `root`               | `glob`    | `src/**/*.js`                   |
 | --coverage-output-dir      | Coverage output directory in `root`                   | `string`  | `.nyc_output`                   |
+| --clear-coverage-dir,      | Clear the Coverage output directory before running    |           |                                 |
+| --clr-cov                  | test, when coverage option is enabled                 | `bool`    | `true`                          |
```
Comment (krampstudio) [inline README.md]: I'm ok with the option name  `clear-coverage-dir` but `--clr-cov` looks weird to me. The long form can be sufficient no ?

File: bin/qunit-testrunner.js [hunk 2]
```diff
@@ -99,6 +99,12 @@ yargs
         describe: 'Output of coverage info',
         default: '.nyc_output'
     })
+    .option('clear-coverage-dir', {
+        describe: 'Clear the Coverage output directory before running test, when coverage option is enabled',
```
Comment (krampstudio) [inline bin/qunit-testrunner.js]: typo : `Coverage`  -> `coverage`

File: README.md (hunk 1 above)
Comment (IgorSalevan) [reply README.md]: Sure, we can remove the alias. Just wanted to use a short version because of the long command line in Depp Teachers Portal.

Comment (IgorSalevan) [reply README.md]: Done.

File: bin/qunit-testrunner.js (hunk 2 above)
Comment (IgorSalevan) [reply bin/qunit-testrunner.js]: Done.

**Comment by IgorSalevan** [comment]
> after the modification my coverage directory will be deleted by default. What do you think?

The reason why I propose this new option is appearing of directory clearing function. My code stopped to work because it needed that .nyc stuff.

So maybe we could change the default value to `false`, if @krampstudio doesn't mind?

**Comment by IgorSalevan** [comment]
A branch was renamed to `breaking/'
New PR is: https://github.com/oat-sa/tao-qunit-testrunner-fe/pull/11

**Comment by krampstudio** [comment]
I think it's a feature, the deletion was already by default

**Comment by IgorSalevan** [comment]
> I think it's a feature, the deletion was already by default

I would say that version 0.1.3 was breaking because of a new deletion feature.

**Comment by btamas** [comment]
> I think it's a feature, the deletion was already by default

It is a new feature, so it could not be already there by default. But if we will merge this, the `.nyc` directory will be deleted by default that can cause unexpected behavior. So this is the reason why I said it is a breaking change.

**Pull request closed without merging**